### PR TITLE
updated forced routing table request

### PR DESCRIPTION
**Pull request by Kinju011**

### CODE REVIEW

File: testkit-backend/lib/testkit/backend/messages/requests/forced_routing_table_update.rb [hunk 1]
```diff
@@ -5,8 +5,12 @@ def process
...
       end
 
-      # def to_object
-      # end
+      def to_object
+        fetch(driverId).session(
+          database: database,
+          bookmarks: bookmarks&.map { |bookmark| Neo4j::Driver::Bookmark.from(Array(bookmark)) }
+          )
+      end
```
Comment (klobuczek) [inline testkit-backend/lib/testkit/backend/messages/requests/forced_routing_table_update.rb:13]: in the current form the code is unreachable.

Comment (klobuczek) [reply testkit-backend/lib/testkit/backend/messages/requests/forced_routing_table_update.rb:13]: java testkit-backend does not implement this request so we don't know how to implement it. I'm not sure if the routing table is updated just by creating a session.

**Review by klobuczek — CHANGES_REQUESTED** [review]

File: testkit-backend/lib/testkit/backend/messages/requests/forced_routing_table_update.rb (hunk 1 above)
Comment (Kinju011) [reply testkit-backend/lib/testkit/backend/messages/requests/forced_routing_table_update.rb:13]: okay

**Comment by klobuczek** [comment]
java and python testkit-backends do not implement this request

**Pull request closed without merging**